packages/nextjs/src/config/wrappers/withSentryServerSideGetInitialProps.ts

```diff
@@ -0,0 +1,42 @@
+import { hasTracingEnabled } from '@sentry/tracing';
+import { NextPage } from 'next';
+
+import { callServerSideDataFetcherWithTracingInstrumentation, withErrorInstrumentation } from './wrapperUtils';
+
+type GetInitialProps = Required<NextPage>['getInitialProps'];
+
+/**
+ * Create a wrapped version of the user's exported `getInitialProps` function
+ *
+ * @param origGetInitialProps The user's `getInitialProps` function
+ * @param parameterizedRoute The page's parameterized route
+ * @returns A wrapped version of the function
+ */
+export function withSentryServerSideGetInitialProps(
+  origGetInitialProps: GetInitialProps,
+  parameterizedRoute: string,
+): GetInitialProps {
+  return async function (
+    ...getInitialPropsArguments: Parameters<GetInitialProps>
+  ): Promise<ReturnType<GetInitialProps>> {
+    const [context] = getInitialPropsArguments;
+    const { req, res } = context;
+
+    const errorWrappedGetInitialProps = withErrorInstrumentation(origGetInitialProps);
+
+    if (req && res && hasTracingEnabled()) {
```

Comment:
According to the [nextjs docs](https://nextjs.org/docs/api-reference/data-fetching/get-initial-props#context-object), `req` and `res` will always be included in the context passed to serverside `getInitialProps`, so I don't think we need this check.

Reply:
Done 06d3a0a2dca4686cc0c53cfa5351773a1715d4b4